tests/config.yaml

```diff
@@ -1,12 +1,12 @@
 document: modularity-testing
 version: 1
 name: docker.io/modularitycontainers/mariadb
-modulemd-url: http://pkgs.fedoraproject.org/cgit/modules/mariadb.git/plain/mariadb.yaml?h=f26
+modulemd-url: https://src.fedoraproject.org/modules/mariadb/raw/master/f/mariadb.yaml
 service:
     port: 3036
 module:
     docker:
-        start: "docker run -it --name mariadb_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306"
```

Comment:
Tests start failing because of this change. In the [SQL queries test](https://github.com/container-images/mariadb/blob/master/tests/sqlqueries.py) the DB is referenced by this name to find out IP adress of the container.